pennylane_lightning/src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(pennylane_lightning_tests)
+
+set(CMAKE_CXX_STANDARD 11)
+find_package(OpenMP)
+
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
```

Comment:
As with Pybind11, it may be best to choose a version tag here, rather than a URL. `release-1.11.0` seems to be the latest. If that allows the tests to build and run, I reckon it is fine to use it. Or, if not, choose the tagged version of the URL given.

Reply:
I totally agree with you! I just left it as the default version in https://google.github.io/googletest/quickstart-cmake.html as I didn't know which version is the most suitable.

Reply:
Sure thing. I think once the tests run locally with the latest release (or a previous iteration or two), we can pin it to that version (release-1.11.0).

Reply:
Updated in f0066d3d417c01c0f0f716851f75fc7447dd5ac8